Formula/restview.rb

```diff
@@ -0,0 +1,64 @@
+class Restview < Formula
+  include Language::Python::Virtualenv
+
+  desc "Viewer for ReStructuredText documents that renders them on the fly"
+  homepage "https://github.com/mgedmin/restview"
+  url "https://github.com/mgedmin/restview/archive/2.7.0.tar.gz"
+  sha256 "ab288234da5c6b3f7cd02d6112d0a020822f43423c086089efeeda2ffac5c388"
+
+  head "https://github.com/mgedmin/restview.git", :shallow => false
+
+  depends_on :python if MacOS.version <= :snow_leopard
+
+  resource "bleach" do
+    url "https://files.pythonhosted.org/packages/1e/67/2562affb99e194cb4b0c0b88e661650d065fcf452d1108116a9530ed9cad/bleach-2.0.0.tar.gz"
+    sha256 "b9522130003e4caedf4f00a39c120a906dcd4242329c1c8f621f5370203cbc30"
+  end
+
+  resource "docutils" do
+    url "https://files.pythonhosted.org/packages/84/f4/5771e41fdf52aabebbadecc9381d11dea0fa34e4759b4071244fa094804c/docutils-0.14.tar.gz"
+    sha256 "51e64ef2ebfb29cae1faa133b3710143496eca21c530f3f71424d77687764274"
+  end
+
+  resource "html5lib" do
+    url "https://files.pythonhosted.org/packages/17/ee/99e69cdcefc354e0c18ff2cc60aeeb5bfcc2e33f051bf0cc5526d790c445/html5lib-0.999999999.tar.gz"
+    sha256 "ee747c0ffd3028d2722061936b5c65ee4fe13c8e4613519b4447123fc4546298"
+  end
+
+  resource "pygments" do
+    url "https://files.pythonhosted.org/packages/source/P/Pygments/Pygments-1.5.tar.gz"
+    sha256 "fe183e3886f597e41f8c88d0e53c796cefddc879bfdf45f2915a383060436740"
+  end
+
+  resource "readme_renderer" do
+    url "https://files.pythonhosted.org/packages/b6/a8/f27c15837fcbcb6110bd0f1dfa04b5fae658a1da3c07f186dba89818a613/readme_renderer-17.2.tar.gz"
+    sha256 "9deab442963a63a71ab494bf581b1c844473995a2357f4b3228a1df1c8cba8da"
+  end
+
+  resource "six" do
+    url "https://files.pythonhosted.org/packages/16/d8/bc6316cf98419719bd59c91742194c111b6f2e85abac88e496adefaf7afe/six-1.11.0.tar.gz"
+    sha256 "70e8a77beed4562e7f14fe23a786b54f6296e34344c23bc42f07b15018ff98e9"
+  end
+
+  resource "webencodings" do
+    url "https://files.pythonhosted.org/packages/0b/02/ae6ceac1baeda530866a85075641cec12989bd8d31af6d5ab4a3e8c92f47/webencodings-0.5.1.tar.gz"
+    sha256 "b36a1c245f2d304965eb4e0a82848379241dc04b865afcc4aab16748587e1923"
+  end
+
+  def install
+    virtualenv_install_with_resources
+  end
+
+  test do
+    require "timeout"
+
+    begin
+      Timeout.timeout(10) do
+        system "#{bin}/restview", "--execute",
+        "curl -fsSL https://raw.githubusercontent.com/mgedmin/restview/master/sample.rst"
```

Comment:
Homebrew typically doesn't like `curl` calls fetching external resources. It might be worth staging this as a resource, if that's feasible with how you're using the tool here.

Reply:
Much better – thanks!